libs/dh/metering-point/feature-overview/src/components/measurement/dh-measurements-day.component.ts

```diff
@@ -113,7 +113,7 @@ export class DhMeasurementsDayComponent {
   private locale = inject<WattSupportedLocales>(LOCALE_ID);
   private sum = computed(
     () =>
-      `${this.formatNumber(this.measurements().reduce((acc, x) => acc + (x.current?.quantity ?? 0), 0))} ${this.unit()}`
+      `${this.formatNumber(this.measurements().reduce((acc, x) => Number(acc) + Number(x.current?.quantity ?? 0), 0))} ${this.unit()}`
```

Comment:
suggestion: let's change some of the mock data to reflect the fact that the value can be `number | string | null | undefined`

Reply:
Fixed